packages/devtools_shared/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
-# 6.0.3-wip
+# 6.0.3
 * `CompareMixin` is now generic, implementing `Comparable<T>` instead of
   `Comparable<dynamic>`, and it's operators each therefore accept a `T`
   argument.
 * `SemanticVersion` now mixes in `CompareMixin<SemanticVersion>`, and it's
   `compareTo` method therefore now accepts a `SemanticVersion`.
+* Fix an issue parsing file paths on Mac that could prevent extensions from being detected.
```

Comment:
Any way to test cover the fix?

Reply:
Theres a tracking issue for that: https://github.com/flutter/devtools/issues/6962. First priority is to land fix to Dart SDK to unblock extension authors.